money.go

```diff
@@ -67,22 +67,20 @@ func defaultMarshalJSON(m Money) ([]byte, error) {
 	return buff.Bytes(), nil
 }
 
-// Amount is a datastructure that stores the amount being used for calculations.
-type Amount struct {
-	val int64
-}
+// Amount is a data structure that stores the amount being used for calculations.
+type Amount = int64
```

Comment:
as you not assigning anything `=` is not required.

```suggestion
type Amount int64
```

Reply:
it is not an assignment, it is a "type alias". Without the "=", it is the definition of a new type. In this case it is used as "alias" of "int" to be more explicit on the use.

Reply:
Good point, sorry my bad